SharedUtilities.gs

```diff
@@ -32,8 +32,8 @@ let SPONSOR_EMAIL = "economicsilver@starmail.net";  // Sponsor's email
 // Triggers and Delays 
 let SUBMISSION_WINDOW_MINUTES = 7;
```

Comment:
Since these vars are being used as days, not minutes, rename the vars to be, for example, SUBMISSION_WINDOW_DAYS. They don't represent minutes and this will be confusing later.

Reply:
I used minutes for testing purposes ofc

Reply:
And its easy to use value like (60*24*7)

Reply:
> Since these vars are being used as days, not minutes, rename the vars to be, for example, SUBMISSION_WINDOW_DAYS. They don't represent minutes and this will be confusing later.

Actually we can't simply replace ..MINUTES.. with ..DAYS.. , as getMinutes() and setMinutes() methods are used through all the script, in many places. (otherwise get/setHours() or get/setData should be used)
But with template saying explicitly the deadline date, there are no problems, just express days in minutes (7 days = 10,080 minutes). Or we can indeed change it to getDate after testing. (still I'd keep it for testing purposes)